Mathlib/Control/Basic.lean

```diff
@@ -219,7 +219,8 @@ instance : Monad (Sum.{v, u} e) where
   pure := @Sum.inr e
   bind := @Sum.bind e
 
-instance : LawfulFunctor (Sum.{v, u} e) := by refine' { .. } <;> intros <;> casesm Sum _ _ <;> rfl
+instance : LawfulFunctor (Sum.{v, u} e) := by
+  refine' { .. } <;> intros <;> (try casesm Sum _ _) <;> rfl
```

Comment:
The `casesm Sum _ _` succeeds on some subgoals and fails on some others, so we need to wrap it in a `try`.